features/fields.feature

```diff
@@ -60,19 +52,19 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
   Scenario: Slechts één enkel attribuut wordt gevraagd
     Als een ingeschreven persoon wordt geraadpleegd met fields=geslachtsaanduiding
     Dan wordt attribuut geslachtsaanduiding teruggegeven
-    En wordt geen enkel ander attribuut dan geslachtsaanduiding teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
     En wordt attribuut _links.self teruggegeven
+    En wordt geen enkel ander attribuut dan geslachtsaanduiding teruggegeven
+    En bevat _links geen ander attribuut dan self
     En wordt er geen gerelateerde sub-resource teruggegeven in _embedded
 
   Scenario: Meerdere attributen worden gevraagd
     Als een ingeschreven persoon wordt geraadpleegd met fields=burgerservicenummer,burgerlijkeStaat,geslachtsaanduiding
     Dan wordt attribuut burgerservicenummer teruggegeven
     En wordt attribuut burgerlijkeStaat teruggegeven
     En wordt attribuut geslachtsaanduiding teruggegeven
-    En wordt geen enkel ander attribuut dan burgerservicenummer, burgerlijkeStaat en geslachtsaanduiding teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
     En wordt attribuut _links.self teruggegeven
+    En wordt geen enkel ander attribuut dan burgerservicenummer, burgerlijkeStaat, geslachtsaanduiding en _links.self teruggegeven
```

Comment:
Net als de vorige scenario zou ik het hier willen versimpelen door:

Dan worden alleen attributen burgerservicenummer, burgerlijkeStaat, geslachtsaanduiding en _links.self teruggegeven
En is _embedded leeg

zodat de ontkennende En stappen niet nodig zijn